run_tests.sh

```diff
@@ -1,3 +1,3 @@
 #! /bin/bash
 
-python -m unittest test.test_dao test.test_model test.scraper.test_challonge test.scraper.test_tio test.int_tests.int_test_challonge
+python -m unittest test.test_dao test.test_model test.test_scraper.test_challonge test.test_scraper.test_tio test.int_tests.int_test_challonge
```

Comment:
This script actually shouldn't be used, it's old. You should be run "nosetests" in the top level directory. Sorry I forgot to delete it. Can you update the README to say that?